webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AutomaticReportsCleaner.scala

```diff
@@ -364,7 +367,7 @@ class AutomaticReportsCleaning(
           logger.debug(s"Found maximun run interval: ${x} minutes")
           x
         case eb: EmptyBox =>
-          logger.error((eb ?~! s"Error when trying to get maximun run interval, defaulting to  5 minutes. Error was: ").messageChain)
+          logger.error((eb ?~! s"Error when trying to get maximun run interval, defaulting to 5 minutes. Error was: ").messageChain)
```

Comment:
shouldn't it be
Error was: ${messageChain}
?

Reply:
no, `·eb` is perhaps an `Empty` in the type system even if I know it's a failure, so I have to first change it to failure `eb ?~! "..."` and then call `.messageChain` on it. But as I'm sure it's a failure given implementation, I now that it will be displayed as "... error was: <- more info here"